packages/core/admin/index.js

```diff
@@ -193,11 +194,27 @@ async function createCacheDir({ dir, plugins }) {
   // copy admin core code
   await copyAdmin(cacheDir);
 
-  // Copy app.js
-  const customAdminConfigFilePath = path.join(dir, 'src', 'admin', 'app.js');
+  // Copy app.js or app.tsx if typescript is enabled
+  const customAdminConfigJSFilePath = path.join(dir, 'src', 'admin', 'app.js');
```

Comment:
Why do we use `js` for javascript and `tsx` for typescript instead of `jsx`/`tsx` or `js`/`ts`?

Reply:
Currently in the admin we only use `js` files even for `jsx` so changing this would be a bit more difficult. Also in the tsconfig I have noticed that doing jsx in ts file the compiler or linter was complaining.